build.sbt

```diff
@@ -32,7 +32,8 @@ lazy val scalaPyTensorFlowCross = crossProject(JVMPlatform, NativePlatform)
   .in(file("tensorflow"))
   .settings(
     name := "scalapy-tensorflow-cross",
-    libraryDependencies += "me.shadaj" %%% "scalapy-numpy" % "0.1.0+6-14ca0424",
+    // scalapy-core version should be consistent with what our version of scalapy-numpy uses
+    libraryDependencies += "me.shadaj" %%% "scalapy-numpy" % "0.1.0+8-a5b49ce1",
```

Comment:
As above, `exclude "me.shadaj" %% "scalapy-core"` (or whatever the syntax, I don't remember) + add an explicit dep on  scalapy-core `0.3.0+31-94930a4d` here